### PR TITLE
Removing tags

### PR DESCRIPTION
**Pull request by KrzysztofKowalczyk**

### CODE REVIEW

File: pipe-storage-sqlite/src/main/java/com/tesco/aqueduct/pipe/storage/sqlite/SQLiteStorage.java [hunk 1]
```diff
@@ -46,20 +47,21 @@ private void createEventTableIfExists() {
...
+        try (
+            Connection connection = dataSource.getConnection();
+            PreparedStatement statement = connection.prepareStatement(EventQueries.getReadEvent(typesCount, maxBatchSize))
+        ) {
             int parameterIndex = 1;
             statement.setLong(parameterIndex++, offset);
 
-            for (int i = 0; i < types.size(); i++, parameterIndex++) {
-                statement.setString(parameterIndex, types.get(i));
+            if (types != null) {
```
Comment (adam-yorwerth) [inline pipe-storage-sqlite/src/main/java/com/tesco/aqueduct/pipe/storage/sqlite/SQLiteStorage.java]: Don't think we need this `if` as the check was already done above

File: pipe-storage-sqlite/src/main/java/com/tesco/aqueduct/pipe/storage/sqlite/SQLiteStorage.java [hunk 2]
```diff
@@ -83,29 +85,29 @@ public int calculateRetryAfter(int messageCount) {
     private Message mapRetrievedMessageFromResultSet(ResultSet resultSet) throws SQLException {
         Message retrievedMessage;
         ZonedDateTime time = ZonedDateTime.of(resultSet.getTimestamp("created_utc").toLocalDateTime(),
-            ZoneId.of("UTC"));
+                ZoneId.of("UTC"));
```
Comment (adam-yorwerth) [inline pipe-storage-sqlite/src/main/java/com/tesco/aqueduct/pipe/storage/sqlite/SQLiteStorage.java]: formatting - can params go on a new line each if they don't fit on a single line.

File: pipe-storage-sqlite/src/main/java/com/tesco/aqueduct/pipe/storage/sqlite/SQLiteStorage.java [hunk 3]
```diff
@@ -83,29 +85,29 @@ public int calculateRetryAfter(int messageCount) {
...
-            resultSet.getLong("msg_offset"),
-            time,
-            null,
-            resultSet.getString("data")
+                resultSet.getString("type"),
+                resultSet.getString("msg_key"),
+                resultSet.getString("content_type"),
+                resultSet.getLong("msg_offset"),
+                time,
+                resultSet.getString("data")
```
Comment (adam-yorwerth) [inline pipe-storage-sqlite/src/main/java/com/tesco/aqueduct/pipe/storage/sqlite/SQLiteStorage.java]: less tabs

File: pipe-storage-sqlite/src/main/java/com/tesco/aqueduct/pipe/storage/sqlite/SQLiteStorage.java [hunk 4]
```diff
@@ -149,9 +151,9 @@ public void write(Message message) {
 
     public void compactUpTo(ZonedDateTime zonedDateTime) {
         // We may want a interface - Compactable - for this method signature
-        try(Connection connection = dataSource.getConnection();
-            PreparedStatement statement = connection.prepareStatement(EventQueries.COMPACT)) {
-            statement.setTimestamp(1,Timestamp.valueOf(zonedDateTime.withZoneSameInstant(ZoneId.of("UTC")).toLocalDateTime()));
+        try (Connection connection = dataSource.getConnection();
+             PreparedStatement statement = connection.prepareStatement(EventQueries.COMPACT)) {
+            statement.setTimestamp(1, Timestamp.valueOf(zonedDateTime.withZoneSameInstant(ZoneId.of("UTC")).toLocalDateTime()));
```
Comment (adam-yorwerth) [inline pipe-storage-sqlite/src/main/java/com/tesco/aqueduct/pipe/storage/sqlite/SQLiteStorage.java]: a few spaces out of place on these 3 lines

File: pipe-storage-test/src/main/groovy/com/tesco/aqueduct/pipe/storage/StorageSpec.groovy [hunk 5]
```diff
@@ -77,60 +77,30 @@ abstract class StorageSpec extends Specification {
...
         messages.size() == resultsSize
 
         where:
-        tags | resultsSize
-        [type:["typeX"]] | 0
-        [type:["type-v1"]] | 1
-        [type:["type-v2", "type-v3"]] | 2
-        [type:["type-v1", "type-v3"]] | 2
-        [type:["type-v1", "type-v2", "type-v3"]] | 3
-        [:] | 3
```
Comment (adam-yorwerth) [inline pipe-storage-test/src/main/groovy/com/tesco/aqueduct/pipe/storage/StorageSpec.groovy]: How this formatting ever ended up in the codebase I'll never know...

File: pipe-storage-postgresql/src/main/java/com/tesco/aqueduct/pipe/storage/PostgresqlStorage.java [hunk 6]
```diff
@@ -79,108 +77,63 @@ private long getLatestOffsetMatchingWithConnection(Connection connection, Map<St
...
 
-    private PreparedStatement getMessagesStatement(Connection connection, Map<String, List<String>> tags, long startOffset) {
+    private PreparedStatement getMessagesStatement(Connection connection, List<String> types, long startOffset) {
         try {
             PreparedStatement query;
 
-            if (tags == null || tags.isEmpty()) {
-                query = connection.prepareStatement(getSelectEventsWithoutTypeFilteringQuery(false, maxBatchSize));
+            if (types == null || types.isEmpty()) {
+                query = connection.prepareStatement(getSelectEventsWithoutTypeFilteringQuery(maxBatchSize));
```
[the remark this answers is not in the document]
Comment (adam-yorwerth) [reply pipe-storage-postgresql/src/main/java/com/tesco/aqueduct/pipe/storage/PostgresqlStorage.java]: Probably for another PR

File: pipe-storage-memory/src/main/java/com/tesco/aqueduct/pipe/storage/InMemoryStorage.java [hunk 7]
```diff
@@ -35,10 +35,10 @@ public InMemoryStorage(int limit, long retryAfter) {
      * Complexity: O(log(n)+limit)
      */
     @Override
-    public MessageResults read(Map<String, List<String>> tags, long offset) {
+    public MessageResults read(List<String> types, long offset) {
```
[the remark this answers is not in the document]
Comment (KrzysztofKowalczyk) [reply pipe-storage-memory/src/main/java/com/tesco/aqueduct/pipe/storage/InMemoryStorage.java]: I would keep it for now, we might want to merge publisher here and allow to have a full pipe running for integration tests for other services to use.

**Pull request merged**